backend/app/store/service/service.go

```diff
@@ -159,6 +159,50 @@ func (s *DataStore) Put(locator store.Locator, comment store.Comment) error {
 	return s.Engine.Update(comment)
 }
 
+// GetStringUserDetail gets string user detail
```

Comment:
I have commented about GetString and SetString aboove. The bottom line - having concrete GetUserEmail/SetUserEmail will be nicer